.github/workflows/test-make-dist.yaml

```diff
@@ -13,6 +13,8 @@ jobs:
 
     steps:
     - run: sudo apt install gettext
+    - name: cython
+      run: sudo pip3 install Cython
```

Comment:
I would like to see this removed. Since ldms can build without Cython, we should not require Cython just to run "make dist". (And even if ldms some day requires Cython, it would be nice to be able to skip such a test to allow us to get to "make dist.) The cython check should be non-fatal and allow configure to complete so that make-dist works when cython is missing. In a build enviroment where we want to guarantee that Cython is included, we would add "--enable-cython" to thte configure command line, and that _would_ error out when Cython is missing.